STYLIZING.md

```diff
@@ -0,0 +1,104 @@
+# Stylizing
+
+You can configure the look and feel of your prompt easily with some built-in
+options. This includes changing foreground and background colors, replacing
+icons and separators, adding newlines...
+**Please be aware that styling options should always be set before loading/sourcing P9K!
+Not every option can be changed at runtime.**
+
+## General Stylizing Options
+
+|Option|Default|Effect|
+|------|-------|------|
+|`P9K_PROMPT_ON_NEWLINE`|`false`|Display the prompt on the next line. Also checkout the sgement `newline`.|
+|`P9K_RPROMPT_ON_NEWLINE`|`false`|Display `RPROMPT` on the same line as `PROMPT` if `P9K_PROMPT_ON_NEWLINE=true`|
+|`P9K_MULTILINE_FIRST_PROMPT_PREFIX_ICON`|`?`|If the prompt is more than one line, this will be the prefix for the first line.|
+|`P9K_MULTILINE_NEWLINE_PROMPT_PREFIX_ICON`|`?`|If the prompt is at least tree lines, this will be the prefix for all lines that are neither the first nor the last.|
```

Comment:
```suggestion
|`P9K_MULTILINE_NEWLINE_PROMPT_PREFIX_ICON`|`├─`|If the prompt is at least tree lines, this will be the prefix for all lines that are neither the first nor the last.|
```

Reply:
Thanks! I forgot I didn't search for the characters when I wrote this...